deepchem/feat/graph_data.py

```diff
@@ -204,6 +205,23 @@ def to_dgl_graph(self, self_loop: bool = False):
 
         return g
 
+    def numpy_to_torch(self):
+        """Convert numpy arrays to torch tensors"""
```

Comment:
Can you expand the docstring and add a usage example?

Reply:
I'd also recommend not doing an in-place transformation. Instead return a new `GraphData` with torch tensors. In general in-place updates can lead to bugs if we lose track of what has been modified. Better to make a fresh clean object